Assets/MixedRealityToolkit.Examples/Demos/Input/Scenes/TurnPointerOnOffExample/TurnPointersOnOff.cs

```diff
@@ -0,0 +1,134 @@
+﻿using Microsoft.MixedReality.Toolkit.Input;
+using Microsoft.MixedReality.Toolkit.UI;
+using Microsoft.MixedReality.Toolkit.Utilities;
+using System;
+using System.Collections.Generic;
+using UnityEngine;
+
+
+namespace Microsoft.MixedReality.Toolkit.Examples.Demos
+{
+    /// <summary>
+    /// This example demonstrates how to turn pointers on and off by 
+    /// specifying custom behaviors.
+    /// </summary>
+    public class TurnPointersOnOff : MonoBehaviour
+    {
+        List<Tuple<Type, Interactable>> pointerToggles = new List<Tuple<Type, Interactable>>();
```

Comment:
Please add a comment on what this tuple is for (i.e. an example of something that it would store)

Reply:
I removed that, in favor of hooking up assets.